### DISCUSSION TITLE
Signing in with PassKeys doesn't work

### DISCUSSION THREAD

**Discussion by mlbuck** [community/community#121188] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I tried a couple of times to create a PassKey.  It allowed me to create a PassKey, but then I couldn't authenticate with the passkey.  I either got an error saying that, "You can't sign in with your passkey" or something not very helpful.  Or it would just spin saying that it was "waiting on the interaction with my browser."  Either way, I have not yet been able to sign in with my passkey.  I tried deleting the passkey, but then I got the former of my two errors above when I tried recreating my passkey.  Any help or direction you can provide would be appreciated.

**Comment by ZFPuhi** [comment]
Hello,

You might wanna check this topic here -> https://github.com/orgs/community/discussions/67791 , as it looks like you aren't the only one that encounters similar issues to the ones described in the topic over the past year.

**Comment by chungweileong94** [comment]
Facing the same issue, however, it works on iOS Safari for some reason. And not sure if this is related, the same thing happens on Microsoft service login, so I'm guessing Microsoft broke something.
